### PR TITLE
[CHK-3223] feat: add mocked case for already paid notice (`PAA_PAGAMENTO_DUPLICATO`)

### PR DESCRIPTION
**Pull request by giovanniberti**
#### List of Changes

 * add mock case for already paid notice (`PAA_PAGAMENTO_DUPLICATO`)

#### Motivation and Context

This PR adds an error case for both _verifyPaymentNoticeV2_ and _activatePaymentNotice_ endpoints for a payment notice that was already paid.
Differently from all other valid payment notices, this new range of payment notices (`30295xxxx...`) cannot be verified or activated successfully: an error of type `PAA_PAGAMENTO_DUPLICATO` with an outcome `KO` will be returned instead.

#### How Has This Been Tested?

Deployed instance on UAT and check with Checkout

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -532,6 +536,11 @@ export async function newExpressApp(
             creditorReferenceId,
           });
           return res.status(paActivate24res[0]).send(paActivate24res[1]);
+        } else if (avvisoPagamentoDuplicato.test(noticenumber)) {
+          const paActivateDuplicatoRes = paActivatePagamentoDuplicato({
+            creditorReferenceId,
```
Comment (cap-ang) [inline src/app.ts:541]: it seems this param is not used

File: src/app.ts [hunk 2]
```diff
@@ -95,12 +97,12 @@ const avviso25 = new RegExp('^30224.*'); // fix response
 const avvisoOver5000 = new RegExp('^30277.*'); // random over 5000 euro + random su 2 transfers
 const avvisoUnder1 = new RegExp('^30288.*'); // random under 1 euro + + random su 2 transfers
 
-const avvisoScaduto = new RegExp('^30299.*'); // PAA_PAGAMENTO_SCADUTO
-
-const avvisoTimeout = new RegExp('^30298.*'); // timeout
-const avvisoErrore = new RegExp('^30297.*'); // paErrorVerify
-
+// Special error cases
+const avvisoPagamentoDuplicato = new RegExp('^30295.*'); // fix response
```
Comment (cap-ang) [inline src/app.ts:101]: is the `fix response` comment a typo?

**Review by cap-ang — APPROVED** [review]
just a few comments, LGTM 👍

File: src/app.ts (hunk 1 above)
Comment (giovanniberti) [reply src/app.ts]: Fixed in 0ce5768

File: src/app.ts (hunk 2 above)
Comment (giovanniberti) [reply src/app.ts]: Fixed in 0ce5768

**Review by cap-ang — APPROVED** [review]
👍

**Pull request merged**